### PR TITLE
Create template for PRs

### PR DESCRIPTION
**Pull request by Aram-Anderson**

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Issue Number: 
+
```
Comment (dphilla) [inline PULL_REQUEST_TEMPLATE.md:2]: This looks good--clean!

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+# Description of changes:
+
```
Comment (dphilla) [inline PULL_REQUEST_TEMPLATE.md:4]: excellent!

**Review by dphilla — COMMENTED** [review]
summary

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+# @mentions of the person or team responsible for reviewing proposed changes:
```
Comment (mikeheft) [inline PULL_REQUEST_TEMPLATE.md:5]: Amazing Grammar @Aram-Anderson !!!

**Comment by mikeheft** [comment]
👍

**Pull request merged**